### DISCUSSION TITLE
Possibility to add personal notes to archives (comments)

### DISCUSSION THREAD

**Discussion by Golddouble** [borgbase/vorta#818] [Ideas]
**Is your feature request related to a problem? Please describe.**
Sometimes I want to make a big change in my home, maybe because I install a new program, that also writes or changes something in my home.
In this case it could be a good idea to make a backup before this change. After the backup I have now a new entry in the archive tab. I can see from when this backup is, but I can not see the reason for this backup. I would like to have the possibility to add a note to each "incremental" backup.

**Describe the solution you'd like**
Maybe you can add a further column in the "Archive" tab, for example beside the column "date". This column should be editable. So that we could enter any note in this field. For example:" latest "incremental" backup without the program XY."

**Describe alternatives you've considered**
It should simply be a possibility to make a note for each "incremental" backup somewhere.

**Comment by samuel-w** [comment]
You could probably do this with #570
